session/pingpong/consumer_totals_storage.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package pingpong
+
+import (
+	"sync"
+
+	"github.com/mysteriumnetwork/payments/crypto"
+	"github.com/pkg/errors"
+)
+
+const consumerTotalStorageBucketName = "consumer_promised_totals"
+
+// ConsumerTotalsStorage allows to store total promised amounts for each channel
+type ConsumerTotalsStorage struct {
+	bolt persistentStorage
+	lock sync.Mutex
```

Comment:
Since there is only one lock, what do you think about simply embedding it, so you can `cts.Lock()` and `cts.Unlock()`?

Reply:
Embedding it will make `Lock()` and `Unlock()` publically available, and someone will use it one day outside and will break everything.

Reply:
Good catch